install-manager/src/main/java/cm/aptoide/pt/install_manager/App.kt

```diff
@@ -17,7 +17,7 @@ import cm.aptoide.pt.install_manager.dto.InstallPackageInfo
 interface App<D> {
   val packageName: String
   val installedVersionName: String?
-  val installedVersionCode: Int?
```

Comment:
What is the reason for this change? I believe that an apps version code according to google documentation is a positive integer, meaning we don't actually need a Long value here? Just curious.

Reply:
Google changed the API for this since SDK `28` now it is Long.
https://developer.android.com/reference/android/content/pm/PackageInfo#getLongVersionCode()

Reply:
Good to know, makes sense, thanks.